packages/server/src/user/user.service.ts

```diff
@@ -97,16 +116,25 @@ export class UserService {
   /**
    * Find unique user for a project using email and login.
    *
-   * @param params object includes: `project_id`, `email`, and `password` in plain text
-   * @returns User object
+   * @param params object includes: `project_id` and `email` as string
+   * @param pwd_plain password in plain text
+   * @returns User object if password matches, otherwise return `null`
    */
-  async loginUserByEmail(params: { project_id; email; password }): Promise<USER> {
-    // TODO: Replace find temperary user
-    for (const user of this.users) {
-      if (params.project_id === user.project_id && params.email === user.email && params.password === user.password) {
-        // return dummy user for testing
+  async loginUserByEmail(params: { project_id; email }, pwd_plain: string): Promise<USER> {
+    const user = await this.prisma.uSER.findUniqueOrThrow({
```

Comment:
NIT: It looks like the function will throw an error if the user does not exist, but returns null if the passwords don't match or if there is an internal error. Its probably best to standardize what happens (either always throws an error or always returns null if something doesn't work). The best course of action would probably be to always throw an error.